.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 python:
   - "2.7"
 jdk:
-  - oraclejdk8
+  - oraclejdk7
```

Comment:
Should these tests run on both JDKs?
If so, you could keep the `- oraclejdk8` line.